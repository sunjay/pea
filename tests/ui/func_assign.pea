fn main() {
    // A name `foo` is in scope, but it's not a local variable, so it should not be assignable
    foo = main;

    // This is fine
    let foo = main;
}

fn foo() {}
