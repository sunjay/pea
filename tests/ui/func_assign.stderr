[tests/ui/func_assign.pea:3:5-3:7] error: invalid left-hand side of assignment

error: aborting due to 1 previous error

